app/helpers/results_helper.rb

```diff
@@ -9,19 +9,42 @@ def render_field_item(doc)
         doc[:document][doc[:field]].join(", ").html_safe
   end
 
+  # truncate abstract to 450 characters
+  def render_abstract(doc)
+    doc[:document][doc[:field]][0].truncate(450).html_safe
+  end
   ##
   # Render clean faceted link to items in series
   def render_series_facet_link(doc)
-    series = doc[:document][doc[:field]]
-    collection = doc[:document][Solrizer.solr_name("collection", :displayable)].first
-    links_to_series = []
+    series = doc[:document][Solrizer.solr_name("parent_unittitles", :displayable)]
+   
+    coll = doc[:document][Solrizer.solr_name("collection", :displayable)].first
 
-    series.each do |ser|
-      links_to_series << link_to(ser, add_clean_facet_params_and_redirect([series_facet, ser],[collection_facet, collection]))
+    if doc[:document][Solrizer.solr_name("unittitle", :displayable)].nil?
```

Comment:
@eshadatta I can't conceive of a use case where this conditional is true. If there is one please add it as an Rspec context to the spec for this function.